File: server/Model/Question.cs
namespace Server.Model
{
    public class Question : BaseEntity
    {
        public string Code { get; set; }
        public string Description { get; set; }

        public string AnswerA { get; set; }
        public string AnswerB { get; set; }
        public string AnswerC { get; set; }
        public string AnswerD { get; set; }
        public string Answer { get; set; }
    }
}

File: ClientApp/src/app/model/question.ts
import { BaseBC } from "../bc/base-bc";
import { QuestionBC } from "../bc/question-bc";
import { BaseEntity } from "./base-entity";

export class Question extends BaseEntity<Question> {

    Code?: string;
    Description?: string;
    Answer?: string;
    AnswerA?: string;
    AnswerB?: string;
    AnswerC?: string;
    AnswerD?: string;

    UserChoice: string = '';

    public override className?(): string {
        return 'Question';
    }

    public override newEntity?(): Question {
        return new Question();
    }

    public override getBC?(): BaseBC<Question> {
        return QuestionBC.getInstance();
    }

}